cg/apps/sequencing_metrics_parser/parsers/bcl_convert.py

```diff
@@ -149,16 +159,36 @@ def get_flow_cell_name(self) -> str:
 
     def get_q30_bases_percent_for_sample_in_lane(self, sample_internal_id: str, lane: int) -> float:
         """Return the percent of bases that are Q30 for a sample and lane."""
-        metric: BclConvertQualityMetrics = self.get_metrics_for_sample_and_lane(
-            metrics_list=self.quality_metrics, sample_internal_id=sample_internal_id, lane=lane
+        metric: List[BclConvertQualityMetrics] = self.get_read_pair_metrics_for_sample_and_lane(
+            sample_internal_id=sample_internal_id, lane=lane
         )
-        return metric.q30_bases_percent
+        return self.calculate_read_pair_q30_bases_percent(metric_list=metric)
+
+    @classmethod
```

Comment:
if you're not using the `cls` variable, you might as well make it static
```suggestion
    @staticmethod
```